### PR TITLE
Refs #35448 -- Reverted SQL formatting for test --debug-sql

### PR DESCRIPTION
**Pull request by saJaeHyukc**
# Trac ticket number

[ticket-35445](https://code.djangoproject.com/ticket/35448)

# Branch description
I reverted the [commit(3283120)](https://github.com/django/django/commit/3283120cca5d5eba5c3619612d0de5ad49dcf054) to resolving the issue of the ticket

# Checklist
- [x] This PR targets the `main` branch. 
- [x] The commit message is written in past tense, mentions the ticket number, and ends with a period.
- [x] I have checked the "Has patch" ticket flag in the Trac system.
- [x] I have added or updated relevant tests.
- [x] I have added or updated relevant docs, including release notes if applicable.
- [x] I have attached screenshots in both light and dark modes for any UI changes.

### CODE REVIEW

File: django/test/runner.py [hunk 1]
```diff
@@ -96,9 +94,7 @@ def printErrorList(self, flavour, errors):
             self.stream.writeln(self.separator2)
             self.stream.writeln(err)
             self.stream.writeln(self.separator2)
-            self.stream.writeln(
-                sqlparse.format(sql_debug, reindent=True, keyword_case="upper")
-            )
+            self.stream.writeln(sql_debug)
```
Comment (felixxm) [inline django/test/runner.py:97]: This is not a fix, you simply revert 3283120cca5d5eba5c3619612d0de5ad49dcf054.

Comment (saJaeHyukc) [reply django/test/runner.py:97]: Should I also modify the 4.2 releases note when I revert?

Comment (saJaeHyukc) [reply django/test/runner.py:97]: Thank you, I reverted to the previous commit.

Comment (saJaeHyukc) [reply django/test/runner.py:97]: My apologies for any confusion caused by my oversight. Based on my understanding, here are the points that require task:

1. Ensuring that SQL reserved words are not capitalized.
2. Adding a new line at the end of each query to separate it from the execution time.

Could you kindly confirm if there's anything else I might have missed? Your input is always valued and appreciated!

**Pull request closed without merging**